diagnostic_common_diagnostics/diagnostic_common_diagnostics/cpu_monitor.py

```diff
@@ -86,13 +86,11 @@ def run(self, stat):
 
         return stat
 
-
-def main(args=None):
-    rclpy.init(args=args)
-
+def get_cpu_diagnostics_node() -> Node:
+    """get cpu diagnostics node"""
     # Create the node
     hostname = socket.gethostname()
-    node = Node(f'cpu_monitor_{hostname.replace("-", "_")}')
+    node = Node('cpu_monitor')
 
     # Declare and get parameters
     node.declare_parameter('warning_percentage', 90)
```

Comment:
added get_cpu_diagnostics_node for easy launch from python_launch.py and changed node name to cpu_monitor instead of cpu_monitor_{hostname.replace("-", "_") not to be depended of hostname in yaml file